src/main/webapp/profile.html

```diff
@@ -47,7 +47,8 @@ <h1>Profile</h1>
           </div>
           <div class="form-group">
             <label for="bio">Bio (max 300 characters):</label>
-            <textarea name="bio" class="form-control" placeholder="Enter a couple of words about yourself" id="bio" maxlength="300"></textarea>
+            <textarea name="bio" class="form-control" placeholder="Enter a couple of words about yourself" 
+            id="bio" maxlength="300"></textarea>
```

Comment:
I think this line should be indented four spaces. https://google.github.io/styleguide/htmlcssguide.html#HTML_Line-Wrapping